src/common/logging.cpp

```diff
@@ -156,8 +156,28 @@ const char *otbrErrorString(otbrError aError)
         error = "OpenThread error";
         break;
 
+    case OTBR_ERROR_SMCROUTE:
+        error = "SMCRoute error";
+        break;
+
+    case OTBR_ERROR_NOT_FOUND:
+        error = "Not found";
+        break;
+
+    case OTBR_ERROR_PARSE:
+        error = "Parse error";
+        break;
+
+    case OTBR_ERROR_NOT_IMPLEMENTED:
+        error = "Not implemented";
+        break;
+
+    case OTBR_ERROR_INVALID_ARGS:
+        error = "Invalid arguments";
+        break;
+
     default:
-        assert(false);
+        error = "Unknown";
```

Comment:
Not related to this PR, but I think it's far too strict to raise assertion failure for unable to convert errors to strings.